test/functional/apps/management/_import_objects.js

```diff
@@ -299,36 +290,33 @@ export default function({ getService, getPageObjects }) {
       });
 
       it('should import saved objects linked to saved searches', async function() {
-        await PageObjects.settings.clickKibanaSavedObjects();
         await PageObjects.settings.importFile(
           path.join(__dirname, 'exports', '_import_objects_saved_search.json')
         );
         await PageObjects.settings.checkImportSucceeded();
         await PageObjects.settings.clickImportDone();
 
-        await PageObjects.settings.navigateTo();
-        await PageObjects.settings.clickKibanaSavedObjects();
+        // await PageObjects.settings.navigateTo();
+        // await PageObjects.settings.clickKibanaSavedObjects();
         await PageObjects.settings.importFile(
           path.join(__dirname, 'exports', '_import_objects_connected_to_saved_search.json')
         );
         await PageObjects.settings.checkImportSucceeded();
         await PageObjects.settings.clickImportDone();
-        await PageObjects.settings.waitUntilSavedObjectsTableIsNotLoading();
 
         const objects = await PageObjects.settings.getSavedObjectsInTable();
         const isSavedObjectImported = objects.includes('saved object connected to saved search');
         expect(isSavedObjectImported).to.be(true);
       });
 
       it('should not import saved objects linked to saved searches when saved search does not exist', async function() {
-        await PageObjects.settings.navigateTo();
-        await PageObjects.settings.clickKibanaSavedObjects();
+        // await PageObjects.settings.navigateTo();
+        // await PageObjects.settings.clickKibanaSavedObjects();
```

Comment:
I'm more into removing, but if commented can you add a comment on top?

Reply:
removed.